tests/test_estimation.py

```diff
@@ -740,7 +740,7 @@ def test_using_dataframes_vs_numpy_arrays(self, data_pred2):
         cf = CoxPHFitter()
         cf.fit(data_pred2, 't', 'E')
 
-        X = data_pred2[cf.data.columns]
+        X = data_pred2[cf._train_columns]
```

Comment:
I suggest not defining the `_train_columns` above, and in this test just calling `data_pred2.columns.difference(['t', 'E'])`